backend/fms_core/models/sample.py

```diff
@@ -275,18 +275,32 @@ def add_error(field: str, error: str):
 
         self.normalize()
 
+        # Sample Kind
         sample_kind_choices = (Sample.BIOSPECIMEN_TYPE_NA_CHOICES if self.extracted_from
                                else Sample.BIOSPECIMEN_TYPE_CHOICES)
-        if self.sample_kind.name not in frozenset(c[0] for c in sample_kind_choices):
-            add_error(
-                "sample_kind",
-                (f"Sample Kind name {self.sample_kind.name} not valid for "
-                 f"{' extracted' if self.extracted_from else ''} sample {self.name}"),
-            )
 
-        if self.transferred_from:
-            if self.sample_kind != self.transferred_from.sample_kind:
-                add_error("sample_kind", "Sample kind need to remain the same during transfer")
+        if self.sample_kind_id is not None:
```

Comment:
Might be good to add an error if this is None since we can more easily format those message to be human readable.

Reply:
There will be an error, it is automatically generated by Django with fields having the attribute blank=False (or not having the attribute blank, which results by default to blank=False)